package.json

```diff
@@ -9,18 +9,18 @@
   "dependencies": {
     "babel-runtime": "^6.26.0",
     "bluebird": "^3.5.1",
-    "chalk": "2.3.2",
     "commander": "^2.16.0",
     "debug": "3.1.0",
+    "getopts": "2.1.1",
```

Comment:
Versions are quite out-of-date by now, do you think it would be a good idea to bump them?

Reply:
@kibertoad Yes. Will do shortly! Thanks. 👍

Reply:
@kibertoad Done! :)